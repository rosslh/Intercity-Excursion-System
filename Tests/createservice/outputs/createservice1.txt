CRE 22222 0 00000 service1 20181223
EOS
