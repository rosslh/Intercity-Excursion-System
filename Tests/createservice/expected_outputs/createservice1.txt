CRE 22222 20181223 service1
EOS
